File: 프로그래머스/기초 트레이닝 - Level 0/배열 비교하기-solve.java
import java.util.Arrays;

class Solution {
    public int solution(int[] arr1, int[] arr2) {
        int answer = 0;
        int sumArr1;
        int sumArr2; 
        
        if (arr1.length > arr2.length) {
            answer = 1;
        } else if (arr1.length < arr2.length) {
            answer = -1;
        } else {
            sumArr1 = Arrays.stream(arr1).sum();
            sumArr2 = Arrays.stream(arr2).sum();
            
            if (sumArr1 > sumArr2) {
                answer = 1;
            } else if (sumArr1 < sumArr2) {
                answer = -1;
            }
        }
        
        return answer;
    }
}

File: 프로그래머스/기초 트레이닝 - Level 0/배열 비교하기-solve.py
def solution(arr1, arr2):
    answer = 0
    if len(arr2) > len(arr1):
        answer = -1
    elif len(arr2) < len(arr1):
        answer = 1
    else:
        if sum(arr2) > sum(arr1):
            answer = -1
        elif sum(arr2) < sum(arr1):
            answer = 1
    
    return answer